### DISCUSSION TITLE
Negative margin not showing

### DISCUSSION THREAD

**Discussion by Chayemor** [tailwindlabs/tailwindcss#11396] [Help]
Version: 2.16

I have tried adding the following negative margin rules but none appear to affected the HTML element. When looking at the whole css rendered for the page, there's some random negative margins that work like `-mt-2 `and `-mt-6` but not `-mt-24` for example. 

I have tried these:

``` javascript
- -mt-2
- -mt-6
- -mt-[200px]
- mt-[-200px]
```

I can add the `-mt-2 `and` -mt-6`, anything else just doesn't seem to have a generated CSS rule for it. What could I be doing so wrong?

![Uploading image.png…]()

**Comment by mybearworld** [comment]
I can't replicate this, could you give more information on your project or perhaps give a reproduction? Play: https://play.tailwindcss.com/tyi2k10uE2
